webpack/base.conf.babel.js

```diff
@@ -77,7 +78,17 @@ module.exports = {
       },
       {
         test: /\.s[a|c]ss$/,
-        loader: 'style-loader!css-loader!sass-loader',
+        use: [
+          'style-loader',
+          'css-loader',
+          {
+            loader: 'sass-loader',
+            options: {
+              data: `@import "scss/${MODULE_SCHEME_NAME}.variables.scss";`,
```

Comment:
nice thing, but let’s keep the default variables.scss. for that, let’s declare all the variables within variables.scss with `!default`, so all the variables placed in `schema.variables.scss` will overwrite the default variables, not completely remove them. see this:

https://sass-scss.ru/documentation/sassscript/peremennie_po-umolchaniyu.html